Examples/UseCases.ipynb

```diff
@@ -82,10 +92,16 @@
     }
    ],
    "source": [
-    "from pydkpro import Pipeline, Component  \n",
+    "from pydkpro import Pipeline, Component\n",
+    "\n",
     "p = Pipeline(version=\"2.0.0\", language='en')\n",
     "p.add(Component().clearNlpSegmenter())\n",
+    "\n",
+    "// REC: Is printTagSet not false by default?\n",
```

Comment:
I have added printTagset to demonstrate "how to pass the parameters" But you are right, default value is ```false```. I'll update it to ```True```

Reply:
Btw. considering that ClearNLP is no longer maintained, maybe use other components for the examples - e.g. CoreNLP or OpenNLP?

Reply:
ok Sure!!